### DISCUSSION TITLE
Possible bug or...?

### DISCUSSION THREAD

**Discussion by Natstrodamus** [snipe/snipe-it#9762] [Q&A]
<img width="498" alt="Screen Shot 2021-06-28 at 12 22 57 PM" src="https://user-images.githubusercontent.com/85517279/123692402-9ae6f800-d80b-11eb-8607-b156d516ee29.png">

I'm having trouble just accessing Admin Settings to adjust my asset tags.  I cleared Cache, cleared plists, using different browsers, and I'm a SuperUser. 

Besides simply trying to adjust the sizing for print, but I'm also just trying to generate a number. This is merely on the web based version, and the installation process is overly complicated for a noob like me. I'm an entry level Jr Admin for an enterprise and besides myself, there are two others who deal with other things, for the most part, but I digress. So what gives?

Thanks in advance,

Natstrodamus

**Comment by Bellwether** [comment]
Are you sure you're a Super Admin?  I don't see the "gears icon" in the top right of your screen.  You can access the Super Admin options page at yourdomain.com/public/admin (on a default, self-hosted installation).

<img width="1680" alt="Image 2021-07-05 at 3 06 18 PM" src="https://user-images.githubusercontent.com/254543/124511042-211eb380-dda3-11eb-8592-b3e7e3153503.png">
